client/js/components/procedure/DpSubmitterList.vue

```diff
@@ -22,13 +22,21 @@
       {{ Translator.trans('export') }}
     </a>
 
+    <div class="u-mb text--right">
+      <dp-column-selector
+        :initial-selection="currentSelection"
+        :selectable-columns="selectableColumns"
+        @selection-changed="setCurrentSelection"
+        use-local-storage
+        local-storage-key="submitterList" />
+    </div>
```

Comment:
What if we put both the "export" link and the solumn selector inside one container? With flex  utils we could do it like so:

```suggestion
    <div class="flex flex-items-center">
      <a
        class="u-mb"
        :href="exportSubmitterList">
        <i
          class="fa fa-download"
          aria-hidden="true" />
        {{ Translator.trans('export') }}
      </a>

      <dp-column-selector
        class="flex-item-end"
        :initial-selection="currentSelection"
        :selectable-columns="selectableColumns"
        @selection-changed="setCurrentSelection"
        use-local-storage
        local-storage-key="submitterList" />
    </div>
```

Reply:
good idea, thx! The two elements are placed in the flex-box now https://github.com/demos-europe/demosplan-core/pull/672/commits/dfe689b4efc47dda998f15d21120a65000d4d24e